docs/proposals/service-discovery/README.md

```diff
@@ -138,15 +92,20 @@ This might be a good place to talk about core concepts and how they relate.
 ### Risks and Mitigations
 
 <!--
-What are the risks of this proposal, and how do we mitigate? 
+What are the risks of this proposal, and how do we mitigate?
 
 How will security be reviewed, and by whom?
 
 How will UX be reviewed, and by whom?
 
 Consider including folks who also work outside the SIG or subproject.
 -->
-Adding a `Service` that resolve to a remote cluster will add a network latency of communication between clusters. 
+
+Adding a `Service` that resolve to a remote cluster will add a network latency of communication between clusters.
+
+#### Feature gate
+
+This feature is an experimental feature, so we add a feature gate to control the global enablement.
```

Comment:
Do we need additional feature switches to control the activation of features? I understand that if we don't create a MultiClusterService object with a cross-cluster service discovery type, then we shouldn't use this feature.

Reply:
We already have this feature gate?

Reply:
There should be none at the moment.

Reply:
Ok, I got it, it's not necessary to introduce a feature gate